src/utils/index.js

```diff
@@ -9,21 +9,14 @@
  */
 
 import docTypes from './frontend/form-helper';
-import runNotification from './frontend/notification-helper';
+import { runNotification, clearProcessing } from './frontend/notification-helper';
 import {
   uploadDocument,
   updateDocument,
   getDocuments,
   deleteDocumentContainer,
   deleteDocumentFile,
   setDocAclPermission,
-  generateUsersList,
-  addUserToPod,
-  getUsersFromPod,
-  deleteUserFromPod,
-  generateActivityTTL,
```

Comment:
Should keep this for generating initial active.ttl file.

Reply:
This functionality has been moved to `updateUserActivity` in `model-helpers/User.js